cloud_experts_tutorials/cloud-experts-aws-resource-tagging.adoc

```diff
@@ -0,0 +1,190 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-aws-resource-tagging"]
+= Tutorial: Tag ROSA components with AWS CLI
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-aws-resource-tagging
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-12
+//---
+//date: '2024-01-24T22:07:08.574151'
+//title: Tag ROSA components with AWS CLI
+//aliases: ['/docs/rosa/tagging']
+//tags: ["AWS", "ROSA", "Tagging"]
+//authors:
+//  - Jakub Wolynko
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+[TIP]
+====
+For better resource utilization and environment maintenance correct and meaningful tagging could save a lot of time and money.
+====
+
+Based on the current date January 2024 ROSA does not support custom resource tagging out-of-the-box. If you want to archive it, use link:https://aws.amazon.com/cli[AWS CLI], and a few different AWS APIs.
```

Comment:
not entirely accurate,  ROSA includes setting tags for cluster creation, but it doesn't support day2 changes to those tags.